ocs_ci/deployment/baremetal.py

```diff
@@ -13,4 +27,394 @@ class BAREMETALUPI(Deployment):
+            cmd = "yum install dnsmasq -y"
+            assert self.helper_node_handler.exec_cmd(cmd=cmd), "Failed to install required package"
+
+            # Enable dnsmasq service on boot
+            cmd = "systemctl enable dnsmasq"
+            assert self.helper_node_handler.exec_cmd(cmd=cmd), "Failed to Enable dnsmasq service"
+
+            # Starting dnsmasq service
+            cmd = "systemctl start dnsmasq"
+            assert self.helper_node_handler.exec_cmd(cmd=cmd), "Failed to Start dnsmasq service"
+
+            cmd = f"mkdir -m 755 -p {constants.bm_tftp_base_dir}"
+            assert self.helper_node_handler.exec_cmd(cmd=cmd), ("Failed to create required folder")
+
+            cmd = f"mkdir -m 755 -p {constants.bm_tftp_base_dir}ocs4qe"
+            assert self.helper_node_handler.exec_cmd(cmd=cmd), ("Failed to create required folder")
+
+            cmd = f"mkdir -m 755 -p {constants.bm_tftp_base_dir}ocs4qe/baremetal"
+            assert self.helper_node_handler.exec_cmd(cmd=cmd), ("Failed to create required folder")
+
+            cmd = f"rm -rf {constants.bm_dnsmasq_dir}*"
+            assert self.helper_node_handler.exec_cmd(cmd=cmd), ("Failed to Delete dir")
+
+            # Install syslinux
+            cmd = "yum install syslinux -y"
+            assert self.helper_node_handler.exec_cmd(cmd=cmd), "Failed to install required package"
+
+            # Copy syslinux files to the tftp path
+            cmd = f"cp -ar /usr/share/syslinux/* {constants.bm_tftp_dir}"
+            assert self.helper_node_handler.exec_cmd(cmd=cmd), ("Failed to Copy required files")
+
+            upload_file(
+                self.host,
+                constants.PXE_CONF_FILE,
+                os.path.join(
+                    constants.bm_dnsmasq_dir,
+                    "dnsmasq.pxe.conf"
+                ),
+                self.user,
+                key_file=self.private_key
+            )
+            upload_file(
+                self.host,
+                constants.COMMON_CONF_FILE,
+                os.path.join(
+                    constants.bm_dnsmasq_dir,
+                    "dnsmasq.common.conf"
+                ),
+                self.user,
+                key_file=self.private_key
+            )
+            # Restarting dnsmasq service
+            cmd = "systemctl restart dnsmasq"
+            assert self.helper_node_handler.exec_cmd(cmd=cmd), "Failed to restart dnsmasq service"
+            with open(constants.RHCOS_IMAGES_FILE) as file_stream:
+                rhcos_images_file = yaml.safe_load(file_stream)
+            ocp_version = get_ocp_version()
+            logger.info(rhcos_images_file)
+            image_data = rhcos_images_file[ocp_version]
+            # Download installer_initramfs
+            cmd = f"wget -O {constants.bm_tftp_dir}/rhcos-installer-initramfs.x86_64.img " \
+                  f"{constants.coreos_url_prefix}" \
+                  f"{image_data['installer_initramfs_url']}"
```

Comment:
please break lines with parenthesis. Apply in other places as well

Reply:
Done